Libraries/PushNotificationIOS/RCTPushNotificationManager.m

```diff
@@ -251,4 +251,53 @@ - (void)handleRemoteNotificationsRegistered:(NSNotification *)notification
   }
 }
 
+RCT_EXPORT_METHOD(getScheduledLocalNotifications:(RCTResponseSenderBlock)callback)
+{
+  NSArray<UILocalNotification *> *scheduledLocalNotifications = [UIApplication sharedApplication].scheduledLocalNotifications;
+
+
+  NSMutableArray *formattedScheduledLocalNotifications = [[NSMutableArray alloc] init];
+
+  for (UILocalNotification *notification in scheduledLocalNotifications) {
+
+    NSMutableDictionary* formattedScheduledLocalNotification = [NSMutableDictionary dictionary];
+
+    if (notification.fireDate) {
+      NSDateFormatter *formatter = [[NSDateFormatter alloc] init];
+      [formatter setDateFormat:@"yyyy-MM-dd'T'HH:mm:ss.SSSZZZZZ"];
+      NSString *fireDateString = [formatter stringFromDate:notification.fireDate];
+
+      [formattedScheduledLocalNotification setObject: fireDateString forKey:@"fireDate"];
+    }
+
+    if (notification.alertBody) {
+      [formattedScheduledLocalNotification setObject: notification.alertBody forKey:@"alertBody"];
+    }
+
+    if (notification.alertAction) {
+      [formattedScheduledLocalNotification setObject: notification.alertAction forKey:@"alertAction"];
+    }
+
+    if (notification.soundName) {
+      [formattedScheduledLocalNotification setObject: notification.soundName forKey:@"soundName"];
+    }
+
+    if (notification.category) {
+      [formattedScheduledLocalNotification setObject: notification.category forKey:@"category"];
+    }
+
+    if (notification.applicationIconBadgeNumber) {
+      [formattedScheduledLocalNotification setObject: [NSNumber numberWithInteger:notification.applicationIconBadgeNumber] forKey:@"applicationIconBadgeNumber"];
+    }
+
+    if (notification.userInfo) {
+      [formattedScheduledLocalNotification setObject: notification.userInfo forKey:@"userInfo"];
+    }
+
+    [formattedScheduledLocalNotifications addObject:formattedScheduledLocalNotification];
+  }
+
+  callback(@[formattedScheduledLocalNotifications]);
```

Comment:
@javache Right, so I just did a test and when I don't wrap it in another `Array` I only get one of the scheduled notification objects back instead of an `Array` of them. Using this JS: 

```js
PushNotificationIOS.getScheduledLocalNotifications( (notifications) => {
  console.log(notifications);
}
```

So that would return something like:

```js
Object { 
  alertBody: "You should get in touch with Aaron", 
  applicationIconBadgeNumber: 3, 
  soundName: "UILocalNotificationDefaultSoundName", 
  fireDate: "2016-06-24T09:03:20.000-06:00", 
  userInfo: Object }
```

Any ideas?